x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_creation_ui/components/rule_preview/index.tsx

```diff
@@ -270,6 +270,7 @@ const RulePreviewComponent: React.FC<RulePreviewProps> = ({
       {showInvocationCountWarning && (
         <>
           <EuiCallOut
+            announceOnMount={false}
```

Comment:
```suggestion
            announceOnMount
```

Should this one be true?

It's conditionally rendered